logging.js

```diff
@@ -34,14 +36,18 @@ export class ServerLogger {
 			headers: { 'Content-Type': 'application/json' },
 			body: JSON.stringify(logs)
 		};
-		let logger = await this._loggerPromise;
-		const resp = await window.fetch(logger.Endpoint, options);
-		if (resp.status === 410) {
-			// endpoint has expired so force refresh
-			this._loggerPromise = D2L.Logging.GetLogger(true);
-			logger = await this._loggerPromise;
-			options = { ...options, mode: 'no-cors' };
-			window.fetch(logger.Endpoint, options);
+		try {
```

Comment:
I've wrapped these in `try` because they make a bunch of HTTP calls which could fail for any number of reasons. If logging fails I don't want an uncaught error interrupting execution. In that case the logs will just be output to the console.